### DISCUSSION TITLE
Кнопка калькулятор в поле

### DISCUSSION THREAD

**Discussion by Leo-ui-528** [totumonline/totum-ru-issues-and-discussions#265] [Как это сделать?] [answered]
Здравствуйте, подскажите пожалуйста, можно ли и как в поле с цифрами сделать кнопку калькулятор?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Таких возможностей нет, но числовые поля поддерживают относительный ввод.

Если ввести `+ 100` число увеличится на `100`. `+-100` — уменьшится.

Также поддерживается:

+ `*` — умножение.
+ `/` — деление.
+ `+10%` — процентная операция.

**Comment by Leo-ui-528** [comment]
Понял, спасибо
